src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -34,6 +39,7 @@ public ModelManager(ReadOnlyAddressBook addressBook, ReadOnlyUserPrefs userPrefs
         this.addressBook = new AddressBook(addressBook);
         this.userPrefs = new UserPrefs(userPrefs);
         filteredPersons = new FilteredList<>(this.addressBook.getPersonList());
+        undoStack = new Stack<>();
```

Comment:
I am suddenly having another thought of extending this function, if the user accidentally shout the app, the Stack will get cleared and this function wouldn't be functional after second activation so what about we also include the undoStack in the storage file?